generators/app/templates/src/layouts/PublicLayout/public.layout.js

```diff
@@ -36,7 +38,7 @@ const PublicLayout = props => {
               {...{ history, location, match }}
               toolbar={[
                 {
-                  component: () => <LanguageDropdown {...props} />,
+                  component: () => <LanguageDropdown {...{ t, ...props }} />,
```

Comment:
@james-martin-jd can you change ...props to specific props to be clear with props what we are passing to the components